management-api-server/src/main/java/com/datastax/mgmtapi/resources/models/BaseEntity.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.resources.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public abstract class BaseEntity {
```

Comment:
That's what I was thinking of. Nice work.

Reply:
:+1: